NETWORK/NetworkAddPedToSynchronisedScene.md

```diff
@@ -10,14 +10,28 @@ void NETWORK_ADD_PED_TO_SYNCHRONISED_SCENE(Ped ped, int netScene, char* animDict
 
 
 ## Parameters
-* **ped**: 
-* **netScene**: 
-* **animDict**: 
-* **animnName**: 
-* **blendInSpeed**: 
-* **blendOutSpeed**: 
-* **duration**: 
-* **flag**: 
-* **playbackRate**: 
-* **p9**: 
+* **ped**: The ped that the animation gets applied to.
+* **netScene**: scene id.
+* **animDict**: The animDict of the animation that you want to play (remember you have to load with RequestAnimDict() before executing this native)
+* **animnName**: The name of the animation that the ped is gonna play.
+* **blendInSpeed**: a float (normal speed is 8.0f).
+* **blendOutSpeed**: a float (normal speed is -8.0f).
+* **duration**: The animation duration (But it doesn't seem to afect anything).
+* **flag**: A big number 100000000 (Also doesn't seem to affect anything).
+* **playbackRate**: Doesn't seem to affect anything.
+* **p9**: Also a big number 100000000 (Doesn't seem to affect anything)
 
+## Animation Dictionaries
+* **Animation List**: https://alexguirre.github.io/animations-list/
+* **Other Option**: More likely to find your desired animation by just searching OpenIV 
+* Most can be found here: (Gta V path)\update\x64\dlcpacks\(Desired folder like mpheist)\dlc.rpf\x64\anim\ingame\clip_anim@.rpf\
+* Most usefull ones start with anim@scripted@...
+
+## Animations
```

Comment:
You should use triple hashes here too, also move above the  `## parameters` section as well.